esp-hal/src/sdio/esp32c6.rs

```diff
@@ -0,0 +1,106 @@
+use esp32c6::{SLC, SLCHOST};
```

Comment:
Don't use the pac directly, use it from `crate::peripherals` instead, this will allow you to target different chips in esp-hal later

Reply:
Thanks, I wasn't aware of the abstractions available, and replaced the direct use with `crate::peripherals`.